.rubocop.yml

```diff
@@ -74,10 +75,10 @@ Style/IndentationWidth:
 Style/IndentHash:
   Severity: fatal
   Enabled: true
+  EnforcedStyle: consistent
 
 Style/InlineComment:
-  Severity: fatal
-  Enabled: true
+  Enabled: false
```

Comment:
Yeah, I'm really not sure _all_ code can be self documenting...  It's a nice goal to have, but there's always something specific or just odd than a comment would really help elucidate.  But if my editor tells me it's bad to add that comment - I quite possibly won't, then it's just a guessing game for the next person who comes along. :(